shared/components/Header/User/AddOfferButton/AddOfferButton.js

```diff
@@ -14,52 +14,54 @@ import { FormattedMessage } from 'react-intl'
 export default class AddOfferButton extends Component {
```

Comment:
Need change name component and this dump component, don't class.

Reply:
SubscribeButton ?